CMakeModules/select_compute_arch.cmake

```diff
@@ -0,0 +1,198 @@
+# Synopsis:
+#   CUDA_SELECT_NVCC_ARCH_FLAGS(out_variable [target_CUDA_architectures])
```

Comment:
Is this file doing what `cuda_compute_capability.cpp` file is doing ?

Reply:
Yes